.github/workflows/style.yml

```diff
@@ -0,0 +1,62 @@
+name: style
+
+# Controls when the action will run. 
+on:
+  push:
+    branches: [ master, dev ]
+  pull_request:
+    branches: [ master, dev ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    container:
+      image: zapatacomputing/z-quantum-default:latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          pip install .
+
+      - name: Python Code Quality and Lint
+        # You may pin to the exact commit or the version.
+        # uses: ricardochaves/python-lint@32032eca67291cd71f88d79e61bc4b904ee03265
+        uses: ricardochaves/python-lint@v1.3.0
+        with:
+          # A list of all paths to test
+          python-root-list: src/
+          # Use Pylint
+          use-pylint: false
+          # Use pycodestyle
+          use-pycodestyle: false
+          # Use Flake8
+          use-flake8: true
+          # Use Black
+          use-black: true
```

Comment:
does black run before flake8 or after? I would assume after is better.

Reply:
Black is check only anyway so I don't think it matters